float64[] sequence
---
int32 errorCode
string errorDescription